docker/hbase/Dockerfile

```diff
@@ -94,11 +94,13 @@ RUN mv /tmp/hadoop-etc/* /opt/hadoop/etc/hadoop
 RUN mv /tmp/hbase-etc/* /opt/hbase/conf/
 RUN mv /tmp/nutch-etc/* /opt/apache-nutch-2.x/conf/
 
-
+ENV JAVA_HOME /usr
```

Comment:
I think JAVA_HOME should be 

JAVA_HOME=/usr/lib/jvm/java-7-oracle/

see below

docker exec -it nutch bash -c 'su hduser'
hduser@7f3e7e0c298e:/root$ env | grep JAVA
JAVA_HOME=/usr/lib/jvm/java-7-oracle/

The above was done on purpose on line https://github.com/apache/nutch/blob/2.x/docker/hbase/Dockerfile#L84